src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -210,8 +213,20 @@ create_gtod()
 
     chmod 0755 ./gtod
 
-    TIME_ZERO=`./gtod`
-    export TIME_ZERO
+    # initialize profile
+    PROFILE="bootstrap_0.prof"
+    now=$(./gtod)
+    echo "#time,name,uid,state,event,msg" > "$PROFILE"
+
+    ip=$(ip addr \
+         | grep 'state UP' -A2 \
+         | grep 'inet' \
+         | awk '{print $2}' \
+         | cut -f1 -d'/')
+    printf "%.4f,%s,%s,%s,%s,%s,%s\n" \
+        "$now" "sync_abs" "bootstrap_0" "MainThread" "$PILOT_ID" \
+        "PMGR_ACTIVE_PENDING" "$(hostname):$ip:$now:$now:$now" \
```

Comment:
is this required to repeat `$now` 3 times?

Reply:
The `sync_abs` timestamps are normally derived from an NTP query, and for that we include the query send time, query receive time, and estimated time.  Here we basically fake a perfect NTP service - which is usually fine as we have second granularity only anyway.